<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="CImprovedPanel" name="ImprovedPanel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>431</width>
      <height>361</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Pannello animato</string>
    </property>
    <property name="whatsThis">
     <string>Pannello animato</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#ImprovedPanel{border-width:1px;
border-color: rgb(68, 68, 68);
border-style:inset;
	background-color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="resizeTime">
     <number>700</number>
    </property>
    <widget class="CLightBlueSwitch" name="Light Blue Switch">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>85</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Eyecandy light blue switch</string>
     </property>
     <property name="whatsThis">
      <string>Eyecandy light blue switch</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
    <widget class="CGreenSwitch" name="Green Switch">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>85</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Eyecandy green switch</string>
     </property>
     <property name="whatsThis">
      <string>Eyecandy green switch</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
    </widget>
    <widget class="CImprovedButton" name="ImprovedButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>101</width>
       <height>95</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Pulsante animato</string>
     </property>
     <property name="whatsThis">
      <string>Pulsante animato</string>
     </property>
     <property name="styleSheet">
      <string notr="true">CImprovedButton{
	border-image: url(:/images/sliders/knob_lblue.png);
}</string>
     </property>
     <property name="fixetTextVisibility">
      <bool>true</bool>
     </property>
     <property name="indicatorVisibility">
      <bool>false</bool>
     </property>
     <property name="indicatorSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="indicatorFont">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="pixmapFile">
      <pixmap resource="resources/components_resources.qrc">:/images/sliders/knob_lblue.png</pixmap>
     </property>
     <property name="appliedMask" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="CLightBlueRadio" name="Light Blue Radio">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>210</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Eyecandy light blue Radio</string>
     </property>
     <property name="whatsThis">
      <string>Eyecandy light blue Radio</string>
     </property>
    </widget>
   </widget>
   <widget class="CImprovedStackWidget" name="SlidingPanel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>-30</y>
      <width>500</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Animated sliding panel</string>
    </property>
    <property name="whatsThis">
     <string>Animated sliding panel</string>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(184, 184, 184);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
border-image: url(:/images/sliders/fill_horizontal.png);


}
QSlider::handle:horizontal {
	border-image: url(:/images/sliders/knob_lblue.png);
    width: 30px;
    height:30px;
}
QSlider::add-page:horizontal {

}
QSlider::sub-page:horizontal {
border-image: url(:/images/sliders/fill_horizontal.png);
}
QSlider::handle:pressed {
	border-image: url(:/images/sliders/knob_lblue.png);
    width: 40px;
    height: 40px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="CImprovedPanel" name="ImprovedPanel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>230</width>
      <height>231</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Pannello animato</string>
    </property>
    <property name="whatsThis">
     <string>Pannello animato</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#ImprovedPanel_2{border-width:1px;
border-color: rgb(68, 68, 68);
border-style:inset;
	background-color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="resizeTime">
     <number>700</number>
    </property>
    <property name="pixmapFile">
     <pixmap>:/img/alarm.png</pixmap>
    </property>
    <property name="appliedMask" stdset="0">
     <bool>false</bool>
    </property>
    <widget class="CImprovedRadioButton" name="Improved Radio Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>210</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Improved Radio Button</string>
     </property>
     <property name="whatsThis">
      <string>Improved Radio Button</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="CImprovedLineEdit" name="ImprovedLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>193</width>
      <height>29</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>--</string>
    </property>
    <property name="whatsThis">
     <string>--</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
        border-style: solid;
        border-width: 1px;
        border-radius: 5px;
        border-color: rgb(125,125,125)
    </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="onlyNumbers">
     <bool>false</bool>
    </property>
    <property name="needAttention">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CGreenSwitch</class>
   <extends>QSlider</extends>
   <header>GreenSwitch</header>
  </customwidget>
  <customwidget>
   <class>CImprovedButton</class>
   <extends>QToolButton</extends>
   <header>ImprovedButton</header>
  </customwidget>
  <customwidget>
   <class>CImprovedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ImprovedLineEdit</header>
  </customwidget>
  <customwidget>
   <class>CImprovedPanel</class>
   <extends>QFrame</extends>
   <header>ImprovedPanel</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CImprovedRadioButton</class>
   <extends>QRadioButton</extends>
   <header>ImprovedRadioButton</header>
  </customwidget>
  <customwidget>
   <class>CImprovedStackWidget</class>
   <extends>QStackedWidget</extends>
   <header>ImprovedStackWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CLightBlueRadio</class>
   <extends>QRadioButton</extends>
   <header>LightBlueRadio</header>
  </customwidget>
  <customwidget>
   <class>CLightBlueSwitch</class>
   <extends>QSlider</extends>
   <header>LightBlueSwitch</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
  <include location="resources/components_resources.qrc"/>
 </resources>
 <connections/>
</ui>
